### ISSUE TITLE
mzMine error

### ISSUE THREAD

**Issue by nurul07** [mzmine/mzmine2#842]
### Basic information

* My operating system:
* My MZmine version: 

### What happened

* Please describe what happened and how it differs from what you expected to happen.
* Please include all parameter values for the MZmine modules you used. 
* Please upload screenshots of any errors and your log/mzmine.log file.
* Please upload an MZmine project file that demonstrates the problem you are reporting.

I was using MZMine version 2.53 to analyze ~700 mzML files, generated by LC-Q-TOF and received this error message about java memory.

I have shown this to my IT staff and they recommended I contact you about the issue to see if you had further suggestions. Our IT department asked if there was away to adjust any of the memory usage parameters. Please contact me if there are other questions regarding this inquiry.

![Error in mzMine](https://user-images.githubusercontent.com/45215561/88700106-78150680-d0c5-11ea-85ba-8777c19ddb93.png)

**Comment by photocyte** [comment]
Hi there,

I'd first check your OS memory monitor (e.g. Task Manager on Windows, Activity Monitor on Mac), to see how close you're getting to your system memory (RAM) limit.  If it looks like you don't have any free RAM left on your machine, then you'd need to either: (1) Set the processing parameters within MZmine2 to use less memory, e.g. have a more relaxed noise floor for peak detection, etc., or (2) find a machine with more RAM.  

If you're not coming close to your machine's RAM limit, then it might be how much Java is given.  I believe you can edit the MZmine startup script.  Check the `startMZmine-Windows.bat` file.  The `DEFAULT_JVM_OPTS` line is what you're looking for. By default, it looks like 80% of your machine's RAM is used, so again, your machine's available RAM is going to be the problem.

In short, 700 files is a huge analysis.  You'll likely need to find a high-memory machine (64+ GB of RAM), to work with all these files at once, without very stringent processing parameters (cropping, high noise floor peak detection, etc.)
